### PR TITLE
Added ctdb setup to samba_libs

### PR DESCRIPTION
**Pull request by vdas-redhat**
Adding ctdb setup code to samba library.

### CODE REVIEW

**Comment by vdas-redhat** [comment]
@milindw96 @pranavprakash20 Please review this patch.

File: glustolibs-gluster/glustolibs/gluster/samba_libs.py [hunk 1]
```diff
@@ -22,6 +22,16 @@
...
 from glustolibs.gluster.volume_libs import is_volume_exported
 from glustolibs.gluster.mount_ops import GlusterMount
+from glustolibs.gluster.ctdb_ops import (
+    edit_hook_script,
+    enable_ctdb_cluster,
+    create_nodes_file,
+    create_public_address_file,
+    start_ctdb_service,
+    is_ctdb_status_healthy)
+from glustolibs.gluster.volume_libs import (
```
Comment (pranavprakash20) [inline glustolibs-gluster/glustolibs/gluster/samba_libs.py:32]: Please remove duplicate import of same library, include setup_volume and wait_for_volume_process_to_be_online imports in line:23

File: glustolibs-gluster/glustolibs/gluster/samba_libs.py [hunk 2]
```diff
@@ -435,6 +445,70 @@ def is_winbind_service_running(mnode):
...
+        ret = enable_ctdb_cluster(mnode)
+        if not ret:
+            return False
+        ret = create_nodes_file(mnode, ctdb_nodes)
+        if not ret:
+            return False
+        ret = create_public_address_file(mnode, ctdb_vips)
+        if not ret:
+            return False
+    server_info = all_servers_info
```
Comment (pranavprakash20) [inline glustolibs-gluster/glustolibs/gluster/samba_libs.py:477]: Can't we use the existing variables here? I see server_info and ctdb_config are used only on Line : 480. In that case, can these two statements be removed? 
Please correct me if I got it wrong.

Comment (vdas-redhat) [reply glustolibs-gluster/glustolibs/gluster/samba_libs.py:477]: Makes sense to use existing variables. I will make the changes and submit the patch.

**Review by pranavprakash20 — APPROVED** [review]
LGTM

**Pull request merged**